lib/player.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# ./lib/Player.rb
+
+class Player
+  attr_accessor :id
+  attr_accessor :name
```

Comment:
You should use `attr_writer` or `attr_reader` respectively as opposed to `attr_accessor`. 
by using `attr_accessor` it allows writing that property from outside the class (not good for most situations). That variable can be set to anything outside of class.